cinema/models.py

```diff
@@ -54,6 +54,10 @@ class MovieSession(models.Model):
     movie = models.ForeignKey(Movie, on_delete=models.CASCADE)
     cinema_hall = models.ForeignKey(CinemaHall, on_delete=models.CASCADE)
 
+    @property
+    def taken_places(self):
+        return Ticket.objects.filter(movie_session=self)
+
```

Comment:
Do you need it if you use a nested serializer for this field in your serializer, not the property from a model?